packages/core-api/src/handlers/businesses/schema.ts

```diff
@@ -46,5 +46,13 @@ export const search: object = {
         businessId: Joi.number()
             .integer()
             .min(1),
+        name: Joi.string()
+            .pattern(/^[a-zA-Z0-9_-]+$/)
```

Comment:
If I remember correct the `pattern` function only exists in joi 16, in joi 15 it should be `regex `.

Reply:
My bad, patch incoming.